packages/server/tests/clients/servicesClient.ts

```diff
@@ -264,13 +254,66 @@ export default class ServicesClient {
         }
 
         return new AliasAccount(
-            aliasAccountId,
+            accountId,
             accountInfo.accountId,
             accountInfo.contractAccountId,
             servicesClient,
             privateKey,
             wallet
         );
+    }
+
+    // Creates an account that has 2 keys - ECDSA and a contractId. This is required for calling contract methods that create HTS tokens.
+    // The contractId should be the id of the contract.
+    // The account should be created after the contract has been deployed.
+    async createAccountWithContractIdKey(contractIdString: string, initialBalance = 10, provider = null, requestId?: string) {
```

Comment:
I think the better approach would be to have the token.key hold the contract ID
e.g. to authorize Contract A to freeze/unfreeze Token T, then T.freezeKey should be set to ContractID A at token creation.

Reply:
Agreed in follow up PR to also utilize T.freezeKey = ContractID for increased coverage